README.md

```diff
@@ -11,24 +11,44 @@ This project is for OSIS which integrates [Scality RING](https://www.scality.com
 
 ## Running the Application
 
-### Command to run Docker Image
-A user needs to be logged into docker using URL: registry.scality.com.
-```sh
-    $ docker login -u <username in registry.scality.com> registry.scality.com
-```
-Once logged images can be pulled and run locally from either dev namespace using the hash or production namespace.
+### Commands to run Docker Image
+1. Docker images can be pulled locally from either dev namespace using the hash or production namespace.
```

Comment:
Although this isn't wrong it feels weird that the first step is to pull from a registry instead of building locally. Why not just `docker build .` ( or similar ).

Reply:
For "Running the Application" idea here is to get the docker image generated by eve instead of manually building a docker image. I can add `docker build .` command to Developers guide to generate a docker image locally.

Reply:
Hmm, I feel like that only works if the docker run command is quick and simple. Like how `docker run mongo` just works. Once you require knowing a git hash or some vague released version it almost becomes a burden. To where, `docker build .` will work every time, plus or minus some extra configs that should also be constant.

Reply:
Completed this in e3ae684ae588760d0b37f12fe614eda2c282a644